ZuluContent/Spells/Fourth/FireField.cs

```diff
@@ -62,13 +62,6 @@ private static void DamageMobile(Mobile caster, Mobile target, int damage)
                 caster.RevealingAction();
 
             caster.DoHarmful(target);
-
-            if (target.ShilCheckSkill(SkillName.MagicResist, 30, 25))
```

Comment:
Is this happening somewhere else now?

Reply:
Yep `SpellHelper.Damage()` has a resist check, so this would be doubling on the skill gain.